source/common/grpc/google_async_client_impl.cc

```diff
@@ -83,8 +83,9 @@ GoogleAsyncClientImpl::GoogleAsyncClientImpl(Event::Dispatcher& dispatcher,
       scope_(scope),
       per_stream_buffer_limit_bytes_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(
           config.google_grpc(), per_stream_buffer_limit_bytes, DefaultBufferLimitBytes)),
-      metadata_parser_(
-          Router::HeaderParser::configure(config.initial_metadata(), /*append=*/false)) {
+      metadata_parser_(Router::HeaderParser::configure(
+          config.initial_metadata(),
+          /*append_action=*/envoy::config::core::v3::HeaderValueOption::OVERWRITE_IF_EXISTS)) {
```

Comment:
You can remove `/*append_action=*/` as it's now obvious.

Reply:
Make sense. I removed it.